### ISSUE TITLE
AttributeError: 'Report' object has no attribute '_symbols'

### ISSUE THREAD

**Issue by ciphersimian** [layday/instawow#44]
Seems to be a persistent problem, ran it with `--debug` but no additional info.  I'm still running the version before yesterday's fixes but I will pull down the latest and check it as well.

```
$ ./instawow -p wow1 --debug update
Traceback (most recent call last):
  File "instawow.__main__", line 4, in <module>
  File "click.core", line 829, in __call__
  File "click.core", line 782, in main
  File "click.core", line 1259, in invoke
  File "click.core", line 1066, in invoke
  File "click.core", line 610, in invoke
  File "click.decorators", line 33, in new_func
  File "instawow.cli", line 306, in update
  File "instawow.cli", line 91, in generate_and_exit
  File "instawow.cli", line 84, in generate
AttributeError: 'Report' object has no attribute '_symbols'
SystemError
```

```
$ ./instawow --version
instawow, version 1.13.1
```

**Comment by ciphersimian** [comment]
Huh, to my relative surprise the new version doesn't seem to have this issue:

```
$ ./instawow -p wow1 update
$ ./instawow --version
instawow, version 1.13.2
```

so perhaps this was also related to the bug from yesterday just manifesting differently?

**Comment by layday** [comment]
The issue is present in the latest version.  It does not manifest _because_ you have the latest version.  *instawow* was attempting to access `_symbols` to report that it's outdated.  This would've been caught by the type checker if I'd remembered to set it up on GitHub Actions.

**Comment by layday** [comment]
This will be fixed in the next version. Thank you for the report.

**Comment by layday** [comment]
This is fixed in 1.13.3.  Thanks again.
